src/game/default/g_main.h

```diff
@@ -26,6 +26,11 @@
 
 #ifdef __GAME_LOCAL_H__
 
+#define PLAYING (g_level.match_status & MSTAT_PLAYING)
+#define TIMEOUT (g_level.match_status & MSTAT_TIMEOUT)
+#define COUNTDOWN (g_level.match_status & MSTAT_COUNTDOWN)
+#define WARMUP (g_level.match_status & MSTAT_WARMUP)
+
```

Comment:
Cool. I *think* I'd prefer to see this prefixed with `G_`, but I'm okay with shorthand like this.